factory/api/apiResolverFactory_test.go

```diff
@@ -448,5 +449,24 @@ func TestCreateApiResolver_createScQueryElement(t *testing.T) {
 		require.True(t, strings.Contains(strings.ToLower(err.Error()), "hasher"))
 		require.Nil(t, scQueryService)
 	})
+}
+
+func TestCreateApiResolver_createBlockchainForScQuery(t *testing.T) {
```

Comment:
Not very proud of this unit test.

Question for review: keep it or remove it?

Reply:
It is ok. I would keep the tests :D